src/Core/Services/TypeHelper.cs

```diff
@@ -29,6 +29,7 @@ public static class TypeHelper
             [typeof(char)] = DbType.StringFixedLength,
             [typeof(Guid)] = DbType.Guid,
             [typeof(byte[])] = DbType.Binary,
+            [typeof(TimeSpan)] = DbType.Time,
```

Comment:
This PR seems to have additional Logic handling DateTime and DateTimeOffset, would it be appropriate (or necessary) to add:
```csharp
            [typeof(DateTime)] = DbType.DateTime,
            [typeof(DateTimeOffset)] = DbType.DateTimeOffset,
            [typeof(TimeSpan)] = DbType.Time,
```

Reply:
Not Adding this hard mapping, since System.DateTime is used for both SqldbType.DateTime and SqldbType.DateTime2. So no change required here as seperate logic exist which does the mapping and also Hotchocolate only has Hotchocolate.Types.DateTime for SqlDbType.DateTime/DateTime2/DateTimeOffset.

Reply:
Can you please add this reason as a comment here? It seems like a valid suggestion but has an equally valid reasoning to not do it, so people looking at the code should know the reasing :)